README.rst

```diff
@@ -5,6 +5,15 @@ NVIDIA DALI
 
 This repository contains extra data used in tests for the NVIDIA DALI project.
 
+Usage
```

Comment:
Just a nitpick: maybe a different title of this section? For me `Usage` of DALI extra would be to run the DALI tests. This is more of a instruction o how to clone it, I think.